.github/workflows/update-wpt.yml

```
@@ -0,0 +1,70 @@
+name: WPT update
+
+on:
+  schedule:
+    # Run once a week at 12:00 AM UTC on Sunday.
+    - cron: 0 0 * * *
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  wpt-subsystem-update:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        subsystem:
+          - url
+
+    steps:
+      - uses: actions/checkout@v4@6d193bf28034eafb982f37bd894289fe649468fc
```

```suggestion
      - uses: actions/checkout@6d193bf28034eafb982f37bd894289fe649468fc
```
Can you add a comment with the tag that this commit points to?